R/ttest.R

```diff
@@ -40,10 +42,23 @@
 #' WoJ %>% t_test(autonomy_selection, mu = 3.62)
 #'
 #' @export
+#' @export
 t_test <- function(data, group_var, ...,
                    var.equal = TRUE, paired = FALSE, pooled_sd = TRUE,
                    levels = NULL, case_var = NULL, mu = NULL) {
 
+  # Add warning for the var.equal deprecation
+  if (var.equal != TRUE) {

```

Comment:
To trigger the warning also when it is explicitly set to TRUE, you could check with `missing` here: http://adv-r.had.co.nz/Functions.html